### PR TITLE
RelationAlgebra 1.7.2 and 1.7.3 for Coq 8.10 and 8.11

### PR DESCRIPTION
**Pull request by damien-pous**

### CODE REVIEW

**Comment by damien-pous** [comment]
@chdoc : cf. my email, it fails to find './configure' to decide whether to compile fhrel.v or not
what's the proper way to fix the opam file?

File: released/packages/coq-relation-algebra/coq-relation-algebra.1.7.2/opam [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+maintainer: "Damien Pous <Damien.Pous@ens-lyon.fr>"
+homepage: "http://perso.ens-lyon.fr/damien.pous/ra/"
+license: "LGPL"
+depends: [
+  "ocaml"
+  "coq" {>= "8.10" & < "8.11~"}
+]
+depopts: [ "coq-mathcomp-ssreflect" ]
+build: [
+  ["sh" "-exc" "./configure --%{coq-mathcomp-ssreflect:enable}%-ssr"]
```
Comment (clarus) [inline released/packages/coq-relation-algebra/coq-relation-algebra.1.7.2/opam]: Does the following work? (not tested)
```suggestion
  ["./configure" "--%{coq-mathcomp-ssreflect:enable}%-ssr"]
```

File: released/packages/coq-relation-algebra/coq-relation-algebra.1.7.2/opam [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  [make "-j%{jobs}%"]
+]
+install: [make "install"]
+remove: [ "sh" "-exc" "rm -r '%{lib}%/coq/user-contrib/RelationAlgebra'" ]
```
Comment (clarus) [inline released/packages/coq-relation-algebra/coq-relation-algebra.1.7.2/opam]:
```suggestion
```
Not needed anymore with opam 2.

File: released/packages/coq-relation-algebra/coq-relation-algebra.1.7.3/opam [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  "ocaml"
+  "coq" {>= "8.11"}
+]
+depopts: [ "coq-mathcomp-ssreflect" ]
+build: [
+  ["sh" "-exc" "./configure --%{coq-mathcomp-ssreflect:enable}%-ssr"]
+  [make "-j%{jobs}%"]
+]
+install: [make "install"]
+remove: [ "sh" "-exc" "rm -r '%{lib}%/coq/user-contrib/RelationAlgebra'" ]
```
Comment (clarus) [inline released/packages/coq-relation-algebra/coq-relation-algebra.1.7.3/opam]:
```suggestion
```
Idem

File: released/packages/coq-relation-algebra/coq-relation-algebra.1.7.2/opam (hunk 1 above)
Comment (clarus) [reply released/packages/coq-relation-algebra/coq-relation-algebra.1.7.2/opam]: Hum... it seems there are no `configure` file in the archive or Git repository actully.

**Comment by chdoc** [comment]
Yes, I am trying to see where that file went. It must have been there at some point ....

**Comment by damien-pous** [comment]
> Yes, I am trying to see where that file went. It must have been there at some point ....
indeed, I thought it was a generated file, I didn't see it was in the 1.7.1 release...

**Comment by chdoc** [comment]
Okay, here is the thing: The file is on the v8.9 branch, but never made it to the v8.10 or master branches.

**Comment by damien-pous** [comment]
ok, so I'll just add it to v8.10 and master and redo the releases...

**Comment by chdoc** [comment]
The relationship between v8.9 and master is nontrivial: This branch is 15 commits ahead, 26 commits behind master. So there may be other things missing as well (while some things definitely shouldn't get merged, because they are to cater for differences in the OCaml API)

**Comment by clarus** [comment]
:tada:

**Comment by damien-pous** [comment]
merci!

**Pull request merged**